pages/c/_cluster/index.vue

```diff
@@ -80,8 +83,12 @@ export default {
       STATE,
     ];
 
+    const metricPoller = new Poler(this.loadMetrics, METRICS_POLL_RATE_MS, MAX_FAILURES);
+
+    metricPoller.start();
+
```

Comment:
Same as earlier, in mounted() so that it doesn't poll in SSR.

Reply:
Moved.